pyproject.toml

```diff
@@ -0,0 +1,42 @@
+[tool.poetry]
+name = "fastlane-bot"
+version = "3.1.5"
+description = "Carbon Arbitrage Bot, an open-source arbitrage protocol, allows any user to perform arbitrage between Bancor ecosystem protocols and external exchanges and redirect arbitrage profits back to the protocol."
+authors = ["Bancor Network <platon@bancor.network>"]
```

Comment:
author field should be our generic author address

Reply:
services@bancor.network
Does this require opening an account somewhere?